cmd/vulcan-blast-radius/main_test.go

```diff
@@ -0,0 +1,162 @@
+/*
+Copyright 2022 Adevinta
+*/
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+
+	checkstate "github.com/adevinta/vulcan-check-sdk/state"
+	"github.com/adevinta/vulcan-checks/cmd/vulcan-blast-radius/intel"
+	report "github.com/adevinta/vulcan-report"
+)
+
+type intelAPIMock struct {
+	blastRadius func(req intel.BlastRadiusRequest) (intel.BlastRadiusResponse, error)
+}
+
+func (i *intelAPIMock) BlastRadius(req intel.BlastRadiusRequest) (intel.BlastRadiusResponse, error) {
+	return i.blastRadius(req)
+}
+
+func TestRun(t *testing.T) {
+	type args struct {
+		target         string
+		assetType      string
+		optJSON        string
+		state          checkstate.State
+		intelAPIClient intelAPI
+	}
+	tests := []struct {
+		name      string
+		args      args
+		env       map[string]string
+		wantVulns []report.Vulnerability
+		wantErr   error
+	}{
+		{
+			name: "Happy Path",
+			args: args{
+				target:    "example.com",
+				assetType: "Hostname",
+				state: checkstate.State{
+					ResultData: &report.ResultData{},
+				},
+				intelAPIClient: &intelAPIMock{
+					blastRadius: func(req intel.BlastRadiusRequest) (intel.BlastRadiusResponse, error) {
+						if req.AssetIdentifier != "example.com" {
+							err := fmt.Errorf("expected identifier: %s, got: %s", "example.com", req.AssetIdentifier)
+							return intel.BlastRadiusResponse{}, err
+						}
+						if req.AssetType != "Hostname" {
+							err := fmt.Errorf("expected asset type: %s, got: %s", "Hostname", req.AssetType)
+							return intel.BlastRadiusResponse{}, err
+						}
+						resp := intel.BlastRadiusResponse{
+							Score:    1.0,
+							Metadata: "meta",
+						}
+						return resp, nil
+					},
+				},
+			},
+			wantVulns: []report.Vulnerability{
+				{
+					Summary:         "Blast Radius Score: 1.00",
+					Description:     blastRadiusVuln.Description,
+					Details:         "meta",
+					Labels:          blastRadiusVuln.Labels,
+					Recommendations: blastRadiusVuln.Recommendations,
+				},
+			},
+		},
+
+		{
+			name: "No Intel API specified",
+			args: args{
+				target:    "example.com",
+				assetType: "Hostname",
+				state: checkstate.State{
+					ResultData: &report.ResultData{},
+				},
+				intelAPIClient: nil,
+			},
+			env:     map[string]string{},
```

Comment:
I think `env` is only used here and it is an empty map. Do we really need it?

Reply:
No, we could remove it.